src/api/Shrooms.Presentation.Api/Controllers/CommentController.cs

```diff
@@ -63,6 +63,11 @@ public async Task<IHttpActionResult> CreateComment(NewCommentViewModel comment)
                     notif.Notify(commentCreatedDto, userHubDto);
                 }, GetOrganizationName());
 
+                _asyncRunner.Run<NewMentionNotifier>(notif =>
```

Comment:
Seems like user could be notified two times for same comment - in case when he is already subscribed to comments and when he is mentioned..
Not sure how to solve it easily, perhaps `NewCommentNotifier` and `NewMentionNotifier` should be merged to avoid doubled notifications - in this case it could be possible to check if user wasn't already notified.